mod-orders-storage/schemas/piece.json

```diff
@@ -87,6 +87,16 @@
       "description": "The date associated item is actually received",
       "type": "string",
       "format": "date-time"
+    },
+    "statusUpdatedDate": {
+      "description": "Date when the status of this piece was last updated",
+      "type": "string",
+      "format": "date-time"
+    },
+    "claimProcessingInterval": {
```

Comment:
**Question**
1. Is `claimProcessingInterval` best name here? Maybe `claimProcessingTermInDays` or `claimProcessingLimitInDays`.
2. What is starting point from which we count number of days?

Reply:
1) we already use the ending 'interval' to indicate the number of days before an action.
2) Delay or sent interval expires: Date the piece status was changed to "Claim delayed" + claimProcessingInterval  (number of days) = < current date